src/credentials/CHIPOperationalCredentials.h

```diff
@@ -45,6 +46,20 @@ struct NodeCredential
     uint16_t mLen         = 0;
 };
 
+struct OperationalCredentialSerializable
+{
+    uint16_t mNodeCredentialLen;
+    uint8_t mNodeCredential[kOperationalCertificateMaxSize];
```

Comment:
Sorry, if I am not very familiar with the code but what is the difference between mNodeCredential and mLeafCertificate?